scripts/symbolize.py

```diff
@@ -98,28 +99,36 @@ def get_elf(self, elf_or_uuid):
             if elf:
                 return elf[0]
 
+    def arch_prefix(self, cmd):
+        if self._arch == 'arm':
+            return 'arm-linux-gnueabihf-' + cmd
+        elif self._arch == 'aarch64':
+            return 'aarch64-linux-gnu-' + cmd
+        else:
+            return ''
+
     def spawn_addr2line(self):
         if not self._addr2line:
             elf = self.get_elf(self._bin)
             if not elf:
                 return
-            if self._arch == 'arm':
-                cmd = 'arm-linux-gnueabihf-addr2line'
-            elif self._arch == 'aarch64':
-                cmd = 'aarch64-linux-gnu-addr2line'
-            else:
+            cmd = self.arch_prefix('addr2line')
+            if not cmd:
                 return
             self._addr2line = subprocess.Popen([cmd, '-f', '-p', '-e', elf],
                                                 stdin = subprocess.PIPE,
                                                 stdout = subprocess.PIPE)
 
-    def resolve(self, addr):
+    def subtract_load_addr(self, addr):
         offs = self._load_addr
         if int(offs, 0) > int(addr, 0):
-            return '???'
-        reladdr = '0x{:x}'.format(int(addr, 0) - int(offs, 0))
+            return ''
+        return '0x{:x}'.format(int(addr, 0) - int(offs, 0))
```

Comment:
Not super important and not really part of this PR, but I guess you always expect to get the numbers in hex? In that case the `int(xyz, 0)` can be changed to `int(xyz, 16)` which makes it a bit more clear what kind of string we are dealing with.